src/biokbase/auth.py

```diff
@@ -41,6 +41,48 @@ def __init__(self, info_dict: dict, token: str = None):
         self.token = token if token is not None else info_dict.get("token")
 
 
+class UserRole:
```

Comment:
These utility wrapper classes are basically a way to make it easy to set defaults, and not carry dicts around with semi-random keys and force whoever consumes them to do the `.get("key", "default")` shuffle.

We could add more specific attribute getters, setters, etc., but IMO, YAGNI ™️ .